zebra-chain/Cargo.toml

```diff
@@ -54,7 +53,8 @@ itertools = { version = "0.10.1", optional = true }
 # ZF deps
 ed25519-zebra = "2"
 equihash = "0.1"
-redjubjub = "0.4"
+# TODO: Update to 0.5 release when published
```

Comment:
Can you create an issue for this, literally, 'Update redjubjub to 0.5/next crates.io release' or something like that?

Reply:
Sure! Created https://github.com/ZcashFoundation/zebra/issues/2352 (and also https://github.com/ZcashFoundation/zebra/issues/2353 for librustzcash)